commons-core/src/main/scala/com/avsystem/commons/misc/OptArg.scala

```diff
@@ -10,6 +10,7 @@ object OptArg {
     * This implicit conversion allows you to pass unwrapped values where `OptArg` is required.
     */
   implicit def argToOptArg[A](value: A): OptArg[A] = OptArg(value)
+  implicit def intToOptArgLong(int: Int): OptArg[Long] = OptArg(int)
```

Comment:
You don't seem to use this one. If you want to start adding these, there's a lot more "implicit" conversions to cover even with just numbers.

Reply:
I'm using it in tests. I was hesitant to add this but it became too annoying. Passing `Int`s where `Long` is expected is very common and this promotion isn't caught by `argToOptArg`.

Reply:
I added `Int => OptArg[Double]` too and IMHO it should be enough. This covers most common and safe numeric promotions.

Reply:
Long -> Double could be an option as well, but I generally agree.

Reply:
The fact that scala has `Long => Double` is wrong and don't want to proliferate it.

Reply:
OMG there's `Long => Float` :man_facepalming: